packages/renderer/src/lib/compose/ComposeDetailsLogs.svelte

```diff
@@ -82,8 +82,9 @@ async function fetchComposeLogs() {
       callback(name, `${colouredName} ${padding} | ${content}`);
     };
 
-    // Get the logs for the container
-    await window.logsContainer(container.engineId, container.id, logsCallback);
+    // Get the logs for the container asynchronously / all at the same time
+    // this will update the terminal as the logs come in.
+    window.logsContainer(container.engineId, container.id, logsCallback);
```

Comment:
I think there is a missing catch here
( I think an eslint rule is still not applied but it would fail)